### PR TITLE
Update gaze.md

### PR DESCRIPTION
**Pull request by sostel**
Updated recommendations to address the differences between head and eye gaze. 
Linked "eye gaze" design guidelines.

### CODE REVIEW

File: mixed-reality-docs/gaze.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Gaze
-description: Gaze is the first form of input and is a primary form of targeting within mixed reality. 
+description: Gaze is the first form of input and is a primary form of targeting within Mixed Reality.
```
Comment (jessemcculloch) [inline mixed-reality-docs/gaze.md]: should be lower case mixed reality per marketing, unless used as part of brand like Windows Mixed Reality

File: mixed-reality-docs/gaze.md [hunk 2]
```diff
@@ -12,11 +12,19 @@ keywords: Mixed Reality, Gaze, Interaction, design
 
 # Gaze
 
-**Gaze** is the first form of input and is a primary form of targeting within mixed reality. Gaze tells you where the user is looking in the world and lets you determine their intent. In the real world, you'll typically look at an object that you intend to interact with. This is the same with gaze.
+**Gaze** is the first form of input and is a primary form of targeting within Mixed Reality.
```
Comment (jessemcculloch) [inline mixed-reality-docs/gaze.md]: lower case mixed reality

File: mixed-reality-docs/gaze.md [hunk 3]
```diff
@@ -40,24 +48,30 @@ On HoloLens 2, interactions can be targeted by either the user's head gaze or th
 
 ## Uses of gaze
 
-As a mixed reality developer, you can do a lot with gaze:
+As a Mixed Reality developer, you can do a lot with head or eye gaze:
```
Comment (jessemcculloch) [inline mixed-reality-docs/gaze.md]: lower case mixed reality

File: mixed-reality-docs/gaze.md (hunk 2 above)
Comment (sostel) [reply mixed-reality-docs/gaze.md]: Updated!

**Pull request merged**